recipes/_start.rb

```diff
@@ -11,8 +11,10 @@
 
 # We want to always ensure that the boot-start script is in place on non-windows platforms
 package_version = Gem::Version.new(node['splunk']['package']['version'])
-command = "#{node['splunk']['cmd']} enable boot-start -user #{node['splunk']['user']}"
+command = "#{node['splunk']['cmd']} enable boot-start -user #{node['splunk']['user']} -systemd-unit-file-name #{node['splunk']['systemd_unit_file_name']}"
```

Comment:
It looks like this argument was introduced in 7.2.2.  I think the systemd-unit-file-name argument needs to be appended down on line 18 instead.